src/renamer.fs

```diff
@@ -316,39 +311,46 @@ module private RenamerImpl =
 
         Array.ofList (oneLetterIdentifiers @ twoLettersIdentifiers)
 
-    let rename shaders =
-        // First rename: give a unique id to each variable.
```

Comment:
Replaced the comment with a better name :)

Reply:
:+1: 
Maybe assignUniqueTemporaryIds then